### PR TITLE
Feature 462/add supervisorid to member directory

### PR DESCRIPTION
**Pull request by warnerj-oci**
Add supervisor name to member directory card and add member

### CODE REVIEW

File: web-ui/src/components/member-directory/MemberModal.jsx [hunk 1]
```diff
@@ -105,47 +112,49 @@ const MemberModal = ({ member = {}, open, onSave, onClose }) => {
...
-          <Button
-            onClick={async () => {
-              onSave(editedMember);
-              await updateMember(editedMember);
-            }}
-            color="primary"
-          >
-            Save Member
-          </Button>
+         <Autocomplete
```
Comment (meyerinj) [inline web-ui/src/components/member-directory/MemberModal.jsx:115]: Tabbing appears to be a bit off here.

File: web-ui/src/components/member-directory/MemberSummaryCard.jsx [hunk 2]
```diff
@@ -12,17 +14,33 @@ import SplitButton from "../split-button/SplitButton";
...
-  const [open, setOpen] = useState(false);
+  const [supervisorName, setSupervisorName] = useState();
 
+  const [open, setOpen] = useState(false);
   const handleOpen = () => setOpen(true);
 
   const handleClose = () => setOpen(false);
 
+  // Get Supervisor's name
+  useEffect(() => {
```
Comment (meyerinj) [inline web-ui/src/components/member-directory/MemberSummaryCard.jsx:30]: Are we worried about doing an extra request for each record slowing down page load? This being the member directory page we already have every member loaded into client memory or the supervisor name can be made a field on the response DTO.

**Review by meyerinj — APPROVED** [review]

File: web-ui/src/components/member-directory/MemberModal.jsx (hunk 1 above)
Comment (warnerj-oci) [reply web-ui/src/components/member-directory/MemberModal.jsx:115]: oops! Thanks.

File: web-ui/src/components/member-directory/MemberSummaryCard.jsx (hunk 2 above)
Comment (warnerj-oci) [reply web-ui/src/components/member-directory/MemberSummaryCard.jsx:30]: That's a good point. I was having trouble accessing the member profile on that page, the call was a fallback.  Let me take another look at that.

**Review by zsbrown97 — APPROVED** [review]

**Pull request merged**